sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient_private.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics
+{
+    public partial class TextAnalyticsClient
+    {
+        private const string TextAnalyticsRoute = "/text/analytics/";
+
+        private const string LanguagesRoute = "/languages";
+        private const string EntitiesRoute = "/entities/recognition/general";
+        private const string PiiEntitiesRoute = "/entities/recognition/pii";
+        private const string SentimentRoute = "/sentiment";
+        private const string KeyPhrasesRoute = "/keyPhrases";
+        private const string EntityLinkingRoute = "/entities/linking";
+
+        private const string ShowStats = "showStats";
+        private const string ModelVersion = "model-version";
+
+        #region Detect Language
+        private static async Task<Response<DocumentResultCollection<DetectedLanguage>>> CreateDetectLanguageResponseAsync(Response response, CancellationToken cancellation)
+        {
+            DocumentResultCollection<DetectedLanguage> result = await TextAnalyticsServiceSerializer.DeserializeDetectLanguageResponseAsync(response.ContentStream, cancellation).ConfigureAwait(false);
+            return Response.FromValue(result, response);
+        }
+
+        private static Response<DocumentResultCollection<DetectedLanguage>> CreateDetectLanguageResponse(Response response)
+        {
+            return Response.FromValue(TextAnalyticsServiceSerializer.DeserializeDetectLanguageResponse(response.ContentStream), response);
```

Comment:
Any reason for all the methods below to not use an expression-bodied member (`=>`) instead of `{ return }`?